test/URI/Extra/MultiHostPortPair.purs

```diff
@@ -30,105 +30,111 @@ spec = do
       (Authority.parser options)
       (Authority.print options)
       "//mongo-1,mongo-2"
-      (Authority
-        Nothing
-        [ This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-1")))
-        , This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-2")))
-        ])
+      { userInfo: Nothing
+      , hosts: [ This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-1")))
+               , This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-2")))
+               ]
```

Comment:
Extremely nitpicky: this would be considered bad formatting, since it's using whitespace alignment. I'd put the array indented on a new line, like:

``` purescript
      , hosts:
          [ This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-1")))
          , This (NameAddress (RegName.unsafeFromString $ nes (SProxy :: SProxy "mongo-2")))
          ]
```

(same goes for all these)